web/libs/editor/tests/integration/e2e/core/info_panels.cy.ts

```diff
@@ -0,0 +1,82 @@
+import { Labels, LabelStudio } from "@humansignal/frontend-test/helpers/LSF";
+import { RichText } from "@humansignal/frontend-test/helpers/LSF/RichText";
+import { FF_DEV_3873 } from "libs/editor/src/utils/feature-flags";
+import { configSimple, dataSimple, resultSimple } from "../../data/core/info_panels";
+
+describe("Label Studio UI info panels", () => {
+  it("Open every panel and interact with regions", () => {
+    LabelStudio.setFeatureFlagsOnPageLoad({
+      [FF_DEV_3873]: true,
+    });
+
+    LabelStudio.init({
+      config: configSimple,
+      task: {
+        annotations: [{ id: 1, result: [resultSimple] }],
+        predictions: [],
+        id: 1,
+        data: dataSimple,
+      },
+    });
+
+    // Info panel is active and it has empty state
+    cy.get("[class$=tab-container_active]").find("#Info-draggable").should("exist");
+    cy.contains("View region details").should("be.visible");
+
+    // Regions panel is also active but has one region in it
+    cy.get("[class$=tab-container_active]").find("#Regions-draggable").should("exist");
+    // This region's score is visible
+    cy.get("[class$=control_type_score]").should("have.text", "0.89");
+
+    // Change regions grouping by openining dropdown; it will be closed automatically
+    cy.contains("Manual").click();
+    cy.contains("Group by Label").should("be.visible");
+    // waiting for dropdown to be fully visible
+    // @todo better options?
+    cy.wait(200);
+    cy.contains("Group by Label").click();
+    cy.get("[class$=lsf-outliner-item__title").contains("Word1").should("be.visible");
```

Comment:
not as the part of this PR, but we need aria-labels for such things 🤔

Reply:
yeah, would be great! we have some helpers, but definitely not enough! added this to a next work list, thank you!